clusterman/cli/status.py

```diff
@@ -116,14 +121,26 @@ def _get_resource_groups_json(
     ]
 
 
-def _status_json(manager: PoolManager, get_node_metadatas: bool) -> StatusJsonObject:
+def _get_migrations(cluster: str, pool: str) -> Collection[MigrationEvent]:
+    """Get node migrations currently in flight for a pool
+
+    :param str cluster: cluster name
+    :param str pool: pool name:
+    :return: collection of migration events
+    """
+    connector = KubernetesClusterConnector(cluster, pool, init_crd=True)
+    return connector.list_node_migration_resources(statuses=[MigrationStatus.PENDING, MigrationStatus.INPROGRESS])
```

Comment:
just talking out loud: would have been nice to make statuses a Set instead of a List so that we enforce uniqueness

Reply:
The argument I'd have against this is pretty much the same I raised in #198 regarding sorting of label selectors, as using sets would end up causing the order of the selector values to randomly change at every call. Considering that at the moment this function is always invoked without user input, so it's pretty easy to verify that there are no duplicates, I'd leave it as it is.